app/setting-persister/setting-persister.js

```diff
@@ -0,0 +1,30 @@
+const Setting = require("../db/setting");
+
+class SettingPersister {
```

Comment:
I saw you've removed `deepCopyObject` (was going to ask why we need it :D). So now my question is, why do you need this class still?

Reply:
Don't ask that, dunno the exact answer. :)

We should store the id of the corresponding element in the db, shouldn't we? As a result, `SettingPersister` holds the logic of communicating with the db separately.

Nevertheless, I'm unsure what'd be the proper solution here. Could you propose some ideas Should we just delete the class wholly and define the related functions somewhere in the code? How'd you store the id in this case(e.g. in a variable)?

Reply:
Why not communicate with `Settings`? What's the difference between `SettingPersister.new(id).copy` and `Setting.findById`? Same goes for update. Also, the setting holds the id in the `_id` field, doesn't it?

Reply:
No, that doesn't hold the id. That's actually just a schema which can be instantiated so that insert a new element into the db - similarly to classes. 

```js
// Sets the default settings.
const newSetting = new Setting();

// The properties are accessible by using the usual dot notation.
newSetting.onEventMessage = something;

// Saving into the DB.
newSetting.save(err => { ... });
```

However, this shouldn't be an obstacle in our case - the id might be stored in a variable.

From my perspective, it was clearer for some reason - there's no reasonable difference between the examples you described above.

Reply:
What about this? https://github.com/c-hive/ws-anywhere/blob/a215f837529ca00308dd424a05713481d1708135/app.js#L23-L25

Reply:
Alright, I was mistaken.